holidays/countries/belize.py

```diff
@@ -0,0 +1,137 @@
+        # holiday; further, if the holiday falls on a Tuesday, Wednesday or
+        # Thursday, the preceding Monday is observed as public holiday
+        name, dt = self._parse_holiday(*args)
+        if self.observed:
+            if self._is_friday(dt) or self._is_sunday(dt):
+                dt = _get_nth_weekday_from(1, MON, dt)
+                name = "%s (Observed)" % name
+            elif (
+                self._is_tuesday(dt)
+                or self._is_wednesday(dt)
+                or self._is_thursday(dt)
+            ):
+                dt = _get_nth_weekday_from(-1, MON, dt)
+                name = "%s (Observed)" % name
+        self._add_holiday(name, dt)
+
+    def _add_holiday(self, *args) -> Optional[date]:
+        # Chapter 289 of the laws of Belize states that if the holiday falls
+        # on a Sunday, the following Monday is observed as public holiday
+        name, dt = self._parse_holiday(*args)
+        if (
+            self.observed
+            and self._is_sunday(dt)
+            and dt + td(days=+1) not in self
+        ):
+            name = "%s (Observed)" % name
+            dt += td(days=+1)
+        return super()._add_holiday(name, dt)
+
+    def _populate(self, year: int) -> None:
+        # Belize was granted independence on 21.09.1981.
+        if year <= 1981:
+            return None
+        super()._populate(year)
+
+        # New Year's Day.
+        self._add_new_years_day("New Year's Day")
+
+        if year >= 2021:
+            # George Price Day.
+            self._add_holiday("George Price Day", JAN, 15)
+
+        # National Heroes and Benefactors Day.
+        self._add_movable_holiday(
+            "National Heroes and Benefactors Day", MAR, 9
+        )
+
+        # Good Friday.
+        self._add_good_friday("Good Friday")
+
+        # Holy Saturday.
+        self._add_holy_saturday("Holy Saturday")
+
+        # Easter Monday.
+        self._add_easter_monday("Easter Monday")
+
+        # Labour Day.
+        self._add_labor_day("Labour Day")
+
+        if year <= 2021:
+            # Commonwealth Day.
+            self._add_movable_holiday("Commonwealth Day", MAY, 24)
+
+        if year >= 2021:
+            # Emancipation Day.
+            self._add_movable_holiday("Emancipation Day", AUG, 1)
+
+        # Saint George's Caye Day.
+        self._add_holiday("Saint George's Caye Day", SEP, 10)
+
+        # Independence Day.
+        self._add_holiday("Independence Day", SEP, 21)
+
+        # Indigenous Peoples' Resistance Day / Pan American Day.
+        self._add_movable_holiday(
+            "Indigenous Peoples' Resistance Day"
+            if year >= 2021
+            else "Pan American Day",
+            date(year, OCT, 12),
+        )
+
+        # Garifuna Settlement Day.
+        self._add_holiday("Garifuna Settlement Day", NOV, 19)
+
+        # Boxing Day.
+        self._add_christmas_day_two("Boxing Day")
```

Comment:
Should be swapped with Christmas day.

Reply:
Boxing Day placed first for right obvserved Christmas calculation (if Christmas falls on Sunday, there should be no observed Christmas on Monday; and in this way, that day is already not empty). I couldn't invent a better way to do it. Only to use `observed_dates` with cycle at the end method.